client/my-sites/site-settings/form-analytics.jsx

```diff
@@ -44,7 +45,13 @@ import QueryJetpackModules from 'components/data/query-jetpack-modules';
 import SettingsSectionHeader from 'my-sites/site-settings/settings-section-header';
 
 const validateGoogleAnalyticsCode = code => ! code || code.match( /^UA-\d+-\d+$/i );
-const hasBusinessPlan = overSome( isBusiness, isEnterprise, isJetpackBusiness, isVipPlan );
+const hasBusinessPlan = overSome(
+	isBusiness,
+	isEcommerce,
+	isEnterprise,
+	isJetpackBusiness,
+	isVipPlan
```

Comment:
Should we  add a trailing comma here? I assume the automated formatter separated it

Reply:
Yes I should have added the trailing comma :)  And yeah the formatter broke it out